dragonfly/implementations/rpi_gpio_provider.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import absolute_import
+
+try:
```

Comment:
sure about just a pass if fail to import? 
GPIO is used in the __init__ of the class...